### ISSUE TITLE
Cannot use task/cgroup in Slurm

### ISSUE THREAD

**Issue by LaHaine** [openhpc/ohpc#476]
With slurm-ohpc-16.05.10-39.1.x86_64 on EL7 I cannot enable the option TaskPlugin=task/cgroup in slurm.conf. I get this error in the log:
slurmstepd[29423]: error: task/cgroup: plugin not compi
led with hwloc support, skipping affinity.

This is the recommended task plugin according to the slurm documentation. It requires slurm to be compiled with the hwloc library.

**Comment by koomie** [comment]
Adding to 1.3.2 milestone.

**Comment by koomie** [comment]
Reproduced locally in v1.3.1 CI environment via following job submission after enabling `task/cgroup` plugin in `slurm.conf` and following options in `cgroup.conf`:

```
ConstrainCores=yes
TaskAffinity=yes
```

```
$ srun -N 2 --ntasks=6 --cpu_bind=cores --pty /bin/bash
slurmstepd: error: task/cgroup: plugin not compiled with hwloc support, skipping affinity.
```

**Comment by koomie** [comment]
I tested the latest slurm build with hwloc-devel addition included from #385, but get the same message. Looking further at the build log: looks like even when hwloc-devel is called out as BuildRequires, slurm configuration is not detecting hwloc:

```
...
[ 185s] configure: WARNING: LZ4 test program build failed.
[  185s] checking for hwloc installation... 
[  185s] configure: WARNING: unable to locate hwloc installation
```
_investigating further...._

**Comment by koomie** [comment]
Had to call out some additional `BuildRequires` and `Requires`, but with the latest build, am able to run a job with the `task/cgroup` plugin enabled.
